pkg/operator/controller/canary/controller.go

```diff
@@ -166,6 +176,21 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		return result, utilerrors.NewAggregate(errors)
 	}
 
+	// Get the canary route rotation annotation value
+	// from the default ingress controller.
+	ic := &operatorv1.IngressController{}
+	if err := r.client.Get(context.TODO(), request.NamespacedName, ic); err != nil {
+		errors = append(errors, fmt.Errorf("failed to get ingress controller %s: %v", ic.Name, err))
```

Comment:
Is `ic.Name` necessarily set if the get failed?  (Doesn't really matter much given that there's only one ingresscontroller that it could be.)

Reply:
Im going to assume it does not get set......fixed. 
Thanks!